CONTRIBUTING.md

```diff
@@ -127,27 +127,82 @@ mvn test
-[Identity Platform](https://cloud.google.com/identity-platform/) project with multi-tenancy
-[enabled](https://cloud.google.com/identity-platform/docs/multi-tenancy-quickstart#enabling_multi-tenancy).
-An existing Firebase project can be upgraded to an Identity Platform project without losing any
-functionality via the
-[Identity Platform Marketplace Page](https://console.cloud.google.com/customer-identity). Note that
-charges may be incurred for active users beyond the Identity Platform free tier.
+tests.
+
+Integration tests are executed against a real life Firebase project. If you do not already
+have one suitable for running the tests against, you can create a new project in the
+[Firebase Console](https://console.firebase.google.com) following the setup guide below.
+Otherwise you can obtain the following credentials from your current project:
+
+
+1. Service account certificate: This can be downloaded as a JSON file from the 
+**'Settings > Service Accounts'** tab of the Firebase console when you click the
+**'Generate new private key'** button. Copy the file into the repo so it's available at
+`integration_cert.json`.
+
+
+2. Web API key: This is displayed in the **'Settings > General'** tab of the Firebase console
+after enabling Authentication as described in the steps below. Copy it and save to a new text
+file at `integration_apikey.txt`.
+
+
+Set up your Firebase project as follows:
+
+
+1. Enable Authentication:
+   1. Go to the Firebase Console, and select **'Authentication'** from the **'Build'** menu.
+   2. Click on **'Get Started'**.
+   3. Select **'Sign-in method > Add new provider > Email/Password'** then enable both the
+   **'Email/Password'** and **'Email link (passwordless sign-in)'** options.
+
+
+2. Enable Firestore:
+   1. Go to the Firebase Console, and select **'Firestore Database'** from the **'Build'** menu.
+   2. Click on the **'Create database'** button. You can choose to set up Firestore either in
+   the production mode or in the test mode.
+
+
+3. Enable Realtime Database:
+   1. Go to the Firebase Console, and select **'Realtime Database'** from the **'Build'** menu.
+   2. Click on the **'Create database'** button. You can choose to set up the Realtime database
+   either in the locked mode or in the test mode.
+   3. In the **'Data'** tab click on the kebab menu (3 dots) and select **'Create Database'**.
+   4. Enter your Project ID (Found in the **'General'** tab in **'Account Settings'**) as the
+   **'Realtime Database reference'**. Again, you can choose to set up the Realtime database
+   either in the locked mode or in the test mode.
+
+
+4. Enable Storage:
+   1. Go to the Firebase Console, and select **'Storage'** from the **'Build'** menu.
+   2. Click on the **'Get started'** button. You can choose to set up the Realtime database
+   either in the production mode or in the test mode.
+
+
+5. Enable the IAM API:
+   1. Go to the[Google Cloud Platform Console](https://console.cloud.google.com)
```

Comment:
Missing a space before the link.

Also, I think they have dropped "Platform" and lowercases the "c" just recently: https://cloud.google.com/compute/docs/console

Reply:
Sounds good